azurerm/resource_arm_servicebus_queue.go

```diff
@@ -117,6 +117,12 @@ func resourceArmServiceBusQueue() *schema.Resource {
 				Optional:   true,
 				Deprecated: "This field has been removed by Azure.",
 			},
+
+			"max_delivery_count": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Default:  10,
```

Comment:
out of interest where does this default value come from? (and if so, are there any validation requirements for this field?)

Reply:
I was able to set it to `999999999` but it does have to be at least `1` so we could go with  `validation.IntAtleast(1)`?

Reply:
@katbyte yes, having it to be at least `1` makes sense to me.